docs/wiki/Known issues.md

```diff
@@ -19,6 +19,31 @@ This section provides an overview of the most impactful limitations and known is
 
 This section outlines known issues that currently affect our modules.
 
+## Microsoft.AAD/DomainServices
+
+The Domain Services module pipeline is expected to fail in our environment for a few reasons:
+
+-  The leveraged service principal has not the required permissions to actually deploy the service in the used tenant
```

Comment:
Should we add which permissions should be set or link to relative MS docs?

Reply:
Added a link to the official docs